glean-core/src/error.rs

```diff
@@ -8,6 +8,8 @@ use ffi_support::{handle_map::HandleError, ExternError};
 
 use rkv::error::StoreError;
 
+use serde_json;
```

Comment:
Is this import necessary? Paths below should in theory work without it.

Reply:
Good point.  Seems to be unnecessary.